specs/protocol/jovian/exec-engine.md

```diff
@@ -83,26 +83,34 @@ contract values to the block builder via `PayloadAttributesV3` parameters.
 
 ## DA Footprint Block Limit
 
-A DA footprint block limit is introduced to limit the total amount of estimated compressed
+A _DA footprint block limit_ is introduced to limit the total amount of estimated compressed
 transaction data that can fit into a block.
- For each transaction, a new resource called DA footprint is tracked, next to its gas usage.
- It is scaled to the gas dimension so that its block total can also be limited by
- the block gas limit, like a block's total gas usage.
+For each transaction, a new resource called DA footprint is tracked, next to its gas usage.
+It is scaled to the gas dimension so that its block total can also be limited by
+the block gas limit, like a block's total gas usage.
 
 Let a block's `daFootprint` be defined as follows:
 
 ```python
-def daFootprint(block)
+def daFootprint(block: Block) -> int:
```

Comment:
Nit: does the `Block` type add much here? Do we need to define it?

Reply:
I was just following the example of EIPs (see e.g. [4844](https://eips.ethereum.org/EIPS/eip-4844#execution-layer-validation)), which specify input and output types. I wanted to add clarity by specifying the output type (`int`), so I found it consistent to also specify the input type.